pyserini/prebuilt_index_info.py

```diff
@@ -1887,6 +1887,398 @@
         "downloaded": False
     },
 
+    # BEIR (v1.0.0) contriever indexes
+    "beir-v1.0.0-trec-covid-contriever": {
+        "description": "Faiss index for BEIR v1.0.0 (TREC-COVID) corpus encoded by Contriever encoder.",
+        "filename": "faiss.beir-v1.0.0-trec-covid.contriever.20230124.tar.gz",
+        "readme": "faiss.contriever.20230124.README.md",
+        "urls": [
+            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/faiss.beir-v1.0.0-trec-covid.contriever.20230124.tar.gz"
+        ],
+        "md5": "5b5baf557979e30e943180627fe31340",
+        "size compressed (bytes)": 488100317,
+        "documents": 171332,
+        "downloaded": False,
+        "texts": "beir-v1.0.0-trec-covid-contriever",
```

Comment:
Is this correct? Shouldn't it refer to a Lucene index?

Reply:
changed all texts from `beir-v1.0.0-{}-contriever` to `beir-v1.0.0-{}.flat`